pkg/kubelet/apis/deviceplugin/v1alpha/api.proto

```diff
@@ -32,6 +32,8 @@ message RegisterRequest {
 	string endpoint = 2;
 	// Schedulable resource name. As of now it's expected to be a DNS Label
 	string resource_name = 3;
+        // Indicates if PreStart call is required before each container start
+	bool pre_start_required = 4;
```

Comment:
I think we will likely add more device plugin options in the future. Should we introduce a DevicePluginOptions message here now and have it contain this boolean config?

Reply:
Sounds like a good idea

Reply:
Done